lib/octokit/client/repositories.rb

```diff
@@ -177,6 +177,19 @@ def delete_repository(repo, options = {})
       end
       alias :delete_repo :delete_repository
 
+      # Transfer repository
+      #
+      # Requires authenticated repository or organization owner of source and destination.
+      #
+      # @see https://developer.github.com/v3/repos/#transfer-a-repository
+      # @param repo [Integer, String, Hash, Repository] A GitHub repository
+      # @return [Sawyer::Resource] Repository info for the transferred repository
+      def transfer_repository(repo, options = {})
```

Comment:
@tarebyte do you think we should make `new_owner` a positional argument here, since it is required?

Reply:
@kytrinyx yup great idea 👍